train.py

```diff
@@ -48,15 +91,20 @@ def main():
         for data in tqdm(loader, leave=False, desc='batch'):
             # Formatted as TxBx[...]
             prv_state = None
+            mask_order = None
             for i in range(cfg.num_frames):
                 inputs = data[i]
                 if prv_state is not None:
                     inputs['prv_state'] = prv_state
                 outputs = model({k: v.to(device) for k, v in inputs.items()})
-                #inputs['motion'] = inputs['scene_flow_3d']
                 prv_state = outputs['state'].detach()
-                loss = motion_loss(outputs['motion'],
-                                   inputs['scene_flow_3d'].to(device))
+                loss_motion = motion_loss(outputs['motion'],
+                                          inputs['scene_flow_3d'].to(device))
+                loss_mask, mask_order = mask_loss(
+                    outputs['logit'],
+                    inputs['mask_3d'].to(device),
+                    mask_order)
+                loss = loss_motion + loss_mask # TODO weight two losses before add
```

Comment:
I think the authors use `1.0 * loss_motion + 5.0 * loss_mask` by default.
Although it's probably better to expose it as a hyperparameter rather than a hardcoded value.

Reply:
Then I will add two constants `1.0` and `5.0` for now. It will be not that hard to make this configurable after the change.